sdk/eventhub/azure-eventhub/azure/eventhub/_common.py

```diff
@@ -182,24 +182,24 @@ def __str__(self):
         event_str += " }"
         return event_str
 
-    def __message_data__(self) -> Dict:
+    def __message_content__(self) -> Dict:
```

Comment:
should this be called out as a breaking change in changelog?

Reply:
I was thinking about this and decided not to, since the user should not be interacting with this method directly. But since it shows up on the APIView, I probably should.